.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug
+assignees: ''
+
+---
+
+Please fill out as much detail as possible:
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```

Comment:
Maybe add a section for "actual behavior". While it could be covered by the repro list or the original description it will be nice to have it in stark contrast to the expected section.